addons/block-palette-icons/addon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Block pallette category icons",
+  "description": "Replaces the colored circles that identify block categories with custom icons.",
+  "tags": ["editor"],
+  "enabledByDefault": false,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*/editor"]
```

Comment:
Change this into `https://scratch.mit.edu/projects/*/editor`. It wouldn't work if you go to a project page, then you see inside. It only works when you link it to the editor view directly.